hypothesis-python/tests/ghostwriter/test_ghostwriter.py

```diff
@@ -426,6 +427,16 @@ def test_unrepr_identity_elem():
             lists(builds(Decimal)),
             {("decimal", "Decimal")},
         ),
+        # find the needed import for from_regex if needed
+        (
+            from_regex(re.compile(".+")),
+            {"re"},
+        ),
+        # but don't add superflous imports
```

Comment:
```suggestion
        # but don't add superfluous imports
```

(I missed this too, which is why we have `codespell` in CI 😅)